roles/pbx/tasks/freepbx.yml

```diff
@@ -118,30 +118,47 @@
     src: "roles/pbx/templates/pbx2.patch"
     dest: "{{ freepbx_install_dir }}/admin/libraries/view.functions.php"
 
-- name: FreePBX - Create /etc/odbc.ini
-  template:
-    src: odbc.ini.j2
-    dest: /etc/odbc.ini
+- name: FreePBX - fix some permissions for nginx
+  file:
+    path: "{{ item.path }}"
+    state: file
+    mode: "{{ item.mode }}"
+  with_items:
+  - { path: '/etc/freepbx.conf', mode: u=rw,g=r,o=r }
+  - { path: '/var/log/asterisk/freepbx.log', mode: u=rw,g=rw,o=rw }
+
+- name: FreePBX - fix some permissions for nginx
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: u=rwx,g=rwx,o=rwx
+  with_items:
+  - /var/www/html/freepbx/admin/assets/less/cache
```

Comment:
Softcode left side of path?  If so using:

```
freepbx_install_dir: /var/www/html/freepbx

Reply:
next round of commits